### PR TITLE
peter-piper

### PR DESCRIPTION
**Pull request by NathanBland**
I also have one suggestion that might be better fitted to your other repo..
But I saw you mentioned in one of your posts that you wanted a code standard.

Might I suggest [standard](https://github.com/feross/standard)?

### CODE REVIEW

**Comment by barberboy** [comment]
Hey, great work! You win all the extra credit points! :100: :100: :100: 

Next, you can help me come up with exercise ideas and write better automated tests :)

I actually planned to use and teach Standard, but unfortunately it doesn't work well with Cloud9's default jshint settings or its beautifier (jsbeautify). For me, that's a no go.

File: peter-piper.js [hunk 1]
```diff
@@ -19,7 +19,14 @@ peterpiper += "Where's the peck of pickled peppers Peter Piper picked?"
...
 var counter = 0;
 
-// TODO: Count the number of times an uppercase P or lowercase p character
-//       occurs in the `peterpiper` string.
-
-console.log("There are " + counter + " P's in Peter Piper.");
\ No newline at end of file
+/*
+var p = peterpiper.match(/p/i) //is this cheating?
+counter = p.length-1 //Caause I feel like this is cheating.
```
Comment (barberboy) [inline peter-piper.js:24]: Definitely NOT cheating! It's also faster than looping through the string characters (you could use something like https://www.npmjs.com/package/spectrophotometer to verify).

**Comment by NathanBland** [comment]
@barberboy I mean... There is a [plugin](https://github.com/feross/standard#text-editor-plugins) for [atom](https://atom.io/) and others. Setting up node isn't that difficult... 

If you wanted i'd even write up a document on how to do it on different OS's. (not OSX though, I don't have access to a system with that, but I bet i could coax @Malachi-M into doing so)

But if you really don't want to leave the comfort of c9 at least maybe make them use the [formatter](https://github.com/feross/standard#is-there-an-automatic-formatter) before submitting it?

**Comment by Malachi-M** [comment]
@NathanBland @barberboy Sure, I'd be willing to help out with such. I will say though that requiring node utilization, while simple after a certain level of knowledge has been attained, may be a stretch. I do not know the constituency of this class so this is simply a conjecture.

**Pull request closed without merging**